pandas/tests/frame/test_deprecations.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import pytest
+
```

Comment:
don't make a new file, just put it in an existing one

Reply:
Any file? There doesn't seem to be a file in which this quite fits.

Reply:
@intEll1gent I think a good place would be where the `to_sparse` functionality itself is tested (but don't know by heart where this is)

Reply:
There seems to be no test for `DataFrame.to_sparse()`? Github search for `test_to_sparse` in `pandas/tests` returns only 1 result, the one for `Series`.